internal/bootfiles/bootfiles_test.go

```diff
@@ -23,6 +24,8 @@ func TestValidator(t *testing.T) {
 	RunSpecs(t, "bootfiles_test")
 }
 
+const FakeServiceBaseURL = "http://192.168.11.22:12345"
```

Comment:
Please use the same `FakeServiceBaseURL` from inventory
https://github.com/openshift/assisted-service/blob/ac59176add3f1a5f89fe47e1df021d9c9ab7589c/internal/bminventory/inventory_test.go#L63
and move it to a common file like https://github.com/openshift/assisted-service/blob/c958b909c9c6e74ba6c7c7db9c7e61b8a93f6257/internal/common/test_configuration.go#L25-L47
instead of duplicating the variable

Reply:
Removed the external API, so no longer relevant :)